aspoet/src/test/kotlin/com/google/androidstudiopoet/models/AndroidModuleBlueprintTest.kt

```diff
@@ -21,15 +22,13 @@ class AndroidModuleBlueprintTest {
 
         val androidModuleBlueprint = getAndroidModuleBlueprint()
 
-        assertOn(androidModuleBlueprint) {
-            activityBlueprints.assertEquals(listOf(ActivityBlueprint(
+        assertThat(androidModuleBlueprint.activityBlueprints).isEqualTo(listOf(ActivityBlueprint(
```

Comment:
I switched this statement to use `Truth` assertions because it helped me debug that fact that the objects had the same `toString()` but were not equal by generating a better error message than the default assertions in junit.